hands-on-exercise/4-run-in-prod/1-run-prod-docker.md

```diff
@@ -1011,6 +1157,99 @@ Note that it has to replace only the first occurrence in the whole.
 
 </CodeGroup>
 
+### CORS
+
+As a last step, you can disable CORS policies so that you are not surprised if you use a node from a Web browser.
+
+1. In `config.toml`:
+
+  <CodeGroup>
+
+  <CodeGroupItem title="TOML">
+
+  ```json [https://github.com/cosmos/b9-checkers-academy-draft/blob/run-prod/prod-sim/node-carol/config/config.toml#L96]
+  cors_allowed_origins = ["*"]
+  ```
+
+  </CodeGroupItem>
+
+  <CodeGroupItem title="One-liner">
+
+  ```sh
+  echo -e node-carol'\n'sentry-alice'\n'sentry-bob'\n'val-alice'\n'val-bob \
+      | xargs -I {} \
+      docker run --rm -i \
+      -v $(pwd)/prod-sim/{}:/root/.checkers \
+      --entrypoint sed \
+      checkersd_i \
+      -Ei 's/^cors_allowed_origins = \[\]/cors_allowed_origins = \["\*"\]/g' \
+      /root/.checkers/config/config.toml
+  ```
+
+  </CodeGroupItem>
+
+  </CodeGroup>
+
+2. In `app.toml`, first location:
+
+
+  <CodeGroup>
+
+  <CodeGroupItem title="TOML">
+
+  ```json [https://github.com/cosmos/b9-checkers-academy-draft/blob/run-prod/prod-sim/node-carol/config/app.toml#L129]
+  enabled-unsafe-cors = true
+  ```
+
+  </CodeGroupItem>
+
+  <CodeGroupItem title="One-liner">
+
+  ```sh
+  echo -e node-carol'\n'sentry-alice'\n'sentry-bob'\n'val-alice'\n'val-bob \
+      | xargs -I {} \
+      docker run --rm -i \
+      -v $(pwd)/prod-sim/{}:/root/.checkers \
+      --entrypoint sed \
+      checkersd_i \
+      -Ei 's/^enabled-unsafe-cors = false/enabled-unsafe-cors = true/g' \
+      /root/.checkers/config/app.toml
+  ```
+
+  </CodeGroupItem>
+
+  </CodeGroup>
+
+1. In `app.toml`, first location:
```

Comment:
@xavierlepretre This repeats the string from Line 1193. Should it read "second location" here?

Reply:
Oops yes, it is "second location".